src/pages/en/guides/markdown-content.mdx

```diff
@@ -583,6 +569,58 @@ const { minutesRead } = Astro.props.frontmatter;
 </html>
 ```
 
+### Extending Markdown config from MDX
+
+Astro's MDX integration will extend [your project's existing Markdown configuration](/en/reference/configuration-reference/#markdown-options) by default. To override individual options, you can specify their equivalent in your MDX configuration.
+
+For example, say you need to disable GitHub-Flavored Markdown and apply a different set of remark plugins for MDX files. You can apply these options like so:
+
+```ts
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  markdown: {
+    syntaxHighlight: 'prism',
+    remarkPlugins: [remarkPlugin1],
+    githubFlavoredMarkdown: true,
+  },
+  integrations: [
+    mdx({
+      // `syntaxHighlight` inherited from Markdown
+
+      // Markdown `remarkPlugins` ignored,
+      // only `remarkPlugin2` applied.
+      remarkPlugins: [remarkPlugin2],
+      // `githubFlavoredMarkdown` overridden to `false`
+      githubFlavoredMarkdown: false,
+    })
+  ]
+});
+```
+
+You may also prefer to disable Markdown config extension. For this, you can set [the `extendMarkdownConfig` option](/en/guides/integrations-guide/mdx/#extendmarkdownconfig) option to `false`:
```

Comment:
My only issue here is making it explicit that you're only disabling Markdown options for MDX files. (Not disabling Markdown for Markdown)

Reply:
I think I made this better?
https://github.com/withastro/docs/pull/2280/commits/9f13d49c3cda1d0285afee6a6d32d5b53a6a3265